native2ika/processor.go

```diff
@@ -45,34 +46,33 @@ func (p *Processor) Run(ctx context.Context) error {
 			if errors.Is(err, ika.ErrEventParsing) {
 				ikaTx := dal.IkaTx{
 					SrID:      sr.ID,
-					Status:    dal.Failed,
+					Status:    int64(dal.Failed),
 					IkaTxID:   txDigest,
 					Timestamp: time.Now().Unix(),
-					Note:      "Transaction successful, but error parsing events.",
+					Note:      sql.NullString{String: "Transaction successful, but error parsing events.", Valid: true},
```

Comment:
why nullstring? Can we avoid null here?

Reply:
Because the field in the table is nullable. We can set is as `NOT NULL` and then we can just insert a empty string `""`